sos/report/plugins/bcache.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Ponnuvel Palaniyappan <ponnuvel.palaniyappan@canonical.com>
```

Comment:
One character too long. Could either split over multi-line or use initials, whichever you prefer.